test/modules/amqp.py

```diff
@@ -40,6 +44,14 @@ def on_message(self, body, message):
     def on_conn_retry(self):
         LOG.error('Retrying connection for {0}'.format(self.__amqp_url))
 
+    def on_connection_error(self, exc, interval):
```

Comment:
If this is called to re-establish a connection, should the __max_error be reset once the connection is established and good again?